ui/apps/platform/src/services/ReportsService.ts

```diff
@@ -244,3 +244,15 @@ export function runReportRequest(
             return response.data;
         });
 }
+
+export function downloadReport(reportId: string) {
+    return axios.get<string>(`/v2/reports/jobs/${reportId}/download`).then((response) => {
+        return response.data;
+    });
+}
+
+export function deleteDownloadableReport(reportId: string) {
+    return axios.delete<Empty>(`/v2/reports/jobs/${reportId}/delete`).then((response) => {
+        return response.data;
+    });
```

Comment:
Often, responses to HTTP DELETE calls don't return any data, just a HTTP response, like 204 "No Content"
(examples, https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE )

A quick check of our API docs shows that our API seems to return 200 but with an empty object `{}` in the body. 

Perusing our FE services, the majority don't parse any response, but some do.

It doesn't look like we are actually accepting this response and passing it on in `useDeleteDownloadModal.ts`

This is more a raise awareness of the topic among the UI team, than a request to change anything.

Reply:
The type for the response is `Empty` so I don't think we really need to do anything here, no? If `response.data` is just empty then it'll return empty anyways. But I do agree with what you mean. Good to keep in mind